velox/exec/Driver.cpp

```diff
@@ -123,7 +123,11 @@ void BlockingState::setResume(std::shared_ptr<BlockingState> state) {
             LOG(ERROR) << "A ContinueFuture for task "
```

Comment:
Do you want to include the error message above into VELOX_FAIL so we don't need the error log?

Reply:
@xiaoxmeng 
Not that I want it, but it can be done. :)